src/test/java/org/fundacionjala/coding/SortTest.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.coding;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ * @author admin-hp
+ */
+public class SortTest {
+    /**
+     * hola.
+     */
+    private Sort so;
+    private String prueba1;
+    private String prueba2;
+    private String prueba3;
+
+
+    /**
+     * declaron las variables para entren por el metodo de invertir.
+     */
+    @Before
+    public void before() {
+        so = new Sort();
+        prueba1 = "sort the inner content in descending order";
+        prueba2 = "wait for me";
+        prueba3 = "this kata is easy";
+    }
+
+    /**
+     * prueba con cadena larga.
+     */
+    @Test
+    public void testSortTheInnerContentVariosC() {
+        assertEquals("srot the inner ctonnet in dsnnieedcg oredr", so.sortTheInnerContent(prueba1));
+    }
+
+    /**
+     * prueba con la cadena mas corta.
+     */
+    @Test
+    public void testSortTheInnerContentVariosc() {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename method "testSortTheInnerContentVariosc" to prevent any misunderstanding/clash with method "testSortTheInnerContentVariosC" defined on line 38. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)